tutorials/assets_pipeline/importing_audio_samples.rst

```diff
@@ -3,19 +3,29 @@
 Importing audio samples
 =======================
 
-Why import?
------------
+Supported files
+---------------
 
-Raw audio data in general is large and undesired. Godot provides two main
-options to import your audio data: WAV and Ogg Vorbis.
+Godot provides three options to import your audio data: WAV, Ogg Vorbis
+and MP3.
 
 Each has different advantages.
 
-* WAV files use raw data or light compression, make few demands on the CPU to play back (hundreds of simultaneous voices in this format are fine), but take up significant space.
-* Ogg Vorbis files use a stronger compression that results in much smaller file size, but require significantly more processing power to play back.
-
-
-.. image:: img/audio_stream_import.png
+* WAV files use raw data or light compression (IMA-ADPCM). They are lightweight
+  on the CPU to play back (hundreds of simultaneous voices in this format are
+  fine). The downside is that they take up a lot of disk space.
+* Ogg Vorbis files use a stronger compression that results in much
+  smaller file size, but require significantly more processing power to
+  play back.
+* MP3 files use better compression than WAV with IMA-ADPCM, but worse than
+  Ogg Vorbis. This means that a MP3 file with roughly equal quality to
+  Ogg Vorbis will be significantly larger. On the bright side, MP3 requires
+  less CPU usage to play back compared to Ogg Vorbis. This makes MP3 useful
+  for mobile and HTML5 projects where CPU resources are limited, especially
+  when playing multiple compressed sounds at the same time (such as long
+  ambient sounds).
+
+.. note:: Godot supports MP3 files with the `minimp3 <https://github.com/lieff/minimp3>`__ decoder.
```

Comment:
This is an implementation detail the user doesn't need to care about, so I'd remove it:

```suggestion
```

Reply:
I wanted something there so if people want to improve the import they know what project to go to. But you're right it doesn't really fit in here.